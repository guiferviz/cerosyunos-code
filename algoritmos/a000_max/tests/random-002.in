2
1
1213997204
1
672624415
