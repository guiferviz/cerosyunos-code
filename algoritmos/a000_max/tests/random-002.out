1213997204
672624415
